opm/autodiff/SimulatorFullyImplicitBlackoil_impl.hpp

```diff
@@ -112,6 +112,7 @@ namespace Opm
         std::vector<int> allcells_;
         const bool has_disgas_;
         const bool has_vapoil_;
+        bool       verbosity_;
```

Comment:
I also prefer the name of this variable to be changed, as for the one in the solver class.